gm4_lightning_rods/data/gm4_lightning_rods/functions/recipe_check.mcfunction

```diff
@@ -1 +1 @@
-execute if score @s gm4_slot_count matches 9 if block ~ ~ ~ dropper{Items:[{Slot:0b,id:"minecraft:blaze_powder"},{Slot:1b,id:"minecraft:blaze_powder"},{Slot:2b,id:"minecraft:blaze_powder"},{Slot:3b,id:"minecraft:blaze_powder"},{Slot:4b,id:"minecraft:stick"},{Slot:5b,id:"minecraft:blaze_powder"},{Slot:6b,id:"minecraft:blaze_powder"},{Slot:7b,id:"minecraft:blaze_powder"},{Slot:8b,id:"minecraft:blaze_powder"}]} run data merge block ~ ~ ~ {Items:[{Slot:8b,id:"blaze_rod",Count:1b,tag:{CustomModelData:1,gm4_custom_crafters:{multiplier:1},gm4_lightning_rod:1b,display:{Lore:['{"translate":"%1$s%3427655$s","with":["Throw for boom!",{"translate":"item.gm4.lightning_rod.lore"}]}'],Name:'{"translate":"%1$s%3427655$s","with":["Lightning Rod",{"translate":"item.gm4.lightning_rod"}],"italic":false}'},HideFlags:1,Enchantments:[{}]}}]}
+execute if score @s gm4_slot_count matches 9 if data storage gm4_custom_crafters:temp/crafter {Items:[{Slot:0b,id:"minecraft:blaze_powder"},{Slot:1b,id:"minecraft:blaze_powder"},{Slot:2b,id:"minecraft:blaze_powder"},{Slot:3b,id:"minecraft:blaze_powder"},{Slot:4b,id:"minecraft:stick"},{Slot:5b,id:"minecraft:blaze_powder"},{Slot:6b,id:"minecraft:blaze_powder"},{Slot:7b,id:"minecraft:blaze_powder"},{Slot:8b,id:"minecraft:blaze_powder"}]} run data merge storage gm4_custom_crafters:temp/crafter {Items:[{Slot:8b,id:"blaze_rod",Count:1b,tag:{CustomModelData:1,gm4_custom_crafters:{multiplier:1},gm4_lightning_rod:1b,display:{Lore:['{"translate":"%1$s%3427655$s","with":["Throw for boom!",{"translate":"item.gm4.lightning_rod.lore"}]}'],Name:'{"translate":"%1$s%3427655$s","with":["Lightning Rod",{"translate":"item.gm4.lightning_rod"}],"italic":false}'},HideFlags:1,Enchantments:[{}]}}]}
```

Comment:
This change will break compressed items as empty Enchantments entries are deleted by some server implementations. The standard way of avoiding this is by the use of a level 0 enchant, such as protection (as implemented before your changes).